www/js/splash/startprefs.js

```diff
@@ -206,7 +206,7 @@ angular.module('emission.splash.startprefs', ['emission.plugin.logger',
             $rootScope.redirectParams = undefined;
             return {state: redirState, params: redirParams};
           } else {
-            return {state: 'root.main.inf_scroll', params: {}};
+            return {state: 'root.main.control', params: {}};
```

Comment:
I don't think this is the correct fix. This is the "default" screen that we use, and it should go to the label screen by default.
`getPendingOnboardingState` is an artifact of a previous version of the app in which people installed the app but then signed up along with their friends using the now defunct habitica integration.

I don't think this is the reason for the profile -> label switch, which is more likely to be an error somewhere else in the codebase.

Let's not include extraneous fixes in here unless they are 100% reproducible given that we are getting to the point where we want to push to production.

Reply:
Comment on the follow-on commit (97ed8f0369c59acb117fdd23f8fa5fd5304eeac0) as well; let's just leave this off for now. I don't want to change the core app functionality unless fixing a 100% broken and reproducible issue because I am afraid that it will introduce additional regressions (like https://github.com/e-mission/e-mission-phone/commit/f863264f19f61ba333c8392ad91c588c94bdff7c did)

Reply:
reverted 93959b540f5e52d67cbb66019e5c5cf5a9c16ee5